easyocr/easyocr.py

```diff
@@ -6,7 +6,7 @@
                    make_rotated_img_list, set_result_with_confidence,\
                    reformat_input_batched, merge_to_free
 from .config import *
-from bidi.algorithm import get_display
+from bidi import get_display
```

Comment:
```suggestion
try:
    from bidi import get_display
except:
    from bidi.algoritm import get_display
```

Reply:
That seems like a good idea to ensure backward compability!